lib/init/config-initializer.js

```diff
@@ -391,7 +394,24 @@ function promptUser() {
             message: "Does your project use TypeScript?",
             enabled: "Yes",
             disabled: "No",
-            initial: 0
+            initial: 0,
+            result(val) {
+                if (val) {
+
+                    // remove airbnb/google javascript style guide, as they does not support ts
+                    styleGuides.push(
+                        { message: "Standard: https://github.com/standard/standard", name: "standard" },
+                        { message: "XO: https://github.com/xojs/eslint-config-xo", name: "xo" }
```

Comment:
Should we in these cases install [eslint-config-standard-with-typescript](https://www.npmjs.com/package/eslint-config-standard-with-typescript) and [eslint-config-xo-typescript](https://www.npmjs.com/package/eslint-config-xo-typescript) rather than configs for JS? (or maybe both JS and TS configs if the TS config doesn't include JS config).